span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/JaegerSpanNormalizer.java

```diff
@@ -139,21 +147,28 @@ public RawSpan convert(Span jaegerSpan) throws Exception {
         jaegerSpan.getTagsList().stream()
             .collect(Collectors.toMap(t -> t.getKey().toLowerCase(), t -> t, (v1, v2) -> v2));
 
-    Optional<String> tenantId = this.tenantIdProvider.getTenantId(tags);
+    Optional<String> maybeTenantId = this.tenantIdProvider.getTenantId(tags);
 
-    if (tenantId.isEmpty()) {
+    if (maybeTenantId.isEmpty()) {
       tenantIdProvider.logWarning(LOG, jaegerSpan);
       return null;
     }
 
+    String tenantId = maybeTenantId.get();
+
+    if (this.tenantIdsToExclude.contains(tenantId)) {
+      LOG.info("Dropping span for tenant id : {}", tenantId);
```

Comment:
better make this warn instead of info .else  too many logs when we exclude a tenant